src/main/java/com/mixpanel/android/viewcrawler/EditProtocol.java

```diff
@@ -52,9 +52,14 @@ public CantGetEditAssetsException(String message, Throwable cause) {
         }
     }
 
-    public EditProtocol(ResourceIds resourceIds, ImageStore imageStore) {
+    public interface OnErrorListener {
```

Comment:
Yes, that's fine- but we should probably define the interface in in ViewCrawler- it's no problem for ViewCrawler to `implements ViewVisitor.OnErrorListener` - particularly since the interface revers to CantVisitException